index.html

```diff
@@ -143,9 +143,12 @@
 
   <div id="bugs" role="main">
     <div id="ba-header" role="log">Results <span id="total"></span> <img id="throbber" alt="Fetching results" src="loading.gif" style="visibility: hidden; float: right"></div>
-    <div id="simple-hint">Do you want to display only simple bugs?
-      <input type="button" value="No" onclick="$(this.parentNode).fadeOut()">
-      <input type="button" value="Yes" onclick="document.getElementById('simple').checked = true; toggleCategoryById('simple');">
+    <div id="simple-hint" class="simple-hint">
```

Comment:
`simple-hint` is still used in some places as an id. For example:
https://github.com/jdm/bugsahoy/blob/master/magic.js#L380

We could modify the code to always expect `simple-hint` to be a class instead of an id, but I don't think it is bad to leave the id for JS and the class for CSS. What do you think?